### PR TITLE
main/postgresql: update secfixes

### PR DESCRIPTION
**Pull request by andypost**
CVE-2019-10127, CVE-2019-10128, CVE-2019-10129, CVE-2019-10130

Ref https://www.postgresql.org/docs/current/release-11-3.html

### CODE REVIEW

File: main/postgresql/APKBUILD [hunk 1]
```diff
@@ -36,6 +36,11 @@ builddir="$srcdir/$pkgname-$pkgver"
 options="!checkroot"
 
 # secfixes:
+#   11.3-r0:
+#   - CVE-2019-10127
+#   - CVE-2019-10128
```
Comment (jirutka) [inline main/postgresql/APKBUILD]: CVE-2019-10127 and CVE-2019-10128 are Windows-only, so this package has never been affected by those.

https://www.postgresql.org/about/news/1939/

Comment (andypost) [reply main/postgresql/APKBUILD]: It was commited as 7cf139bac41c8f2e1885d5f99334daeaeb059ac3 so I updated commit to remove wrong secfixes

**Comment by andypost** [comment]
Tests fails on armv7 with new perl

**Comment by tcely** [comment]
This looks to be merged already. Maybe a new PR for any tweaks would be good?

**Pull request closed without merging**